coolcat-creations/joomla-cms
# Adding label Class for input for Custom fields

Pull Request for Issue #16994. 

### Summary of Changes
Removed the hide Label Option
Replaced hide Label Option with a label Class Option. 
Added B/C Option to add class hidden to the class if label was set previously to hidden.
Added a label class Option for Form
Reordered the Options and added headlines.

See also discussion in  #17177 

### Testing Instructions
1) Hide your label, apply the patch and see if the label is still hidden
2) Add a class for the label in the form and see if it's added
3) Add a class for the label in the output rendering and see if it's added

### Documentation Changes Required
Yes adding the possibility to add a Label Class in the form.

![grafik](https://user-images.githubusercontent.com/828371/32124346-b91dea32-bb67-11e7-8d23-1b5d19ed169a.png)

<img width="784" alt="grafik" src="https://user-images.githubusercontent.com/828371/32124208-435a1d2a-bb67-11e7-88c2-a62e430841c3.png">
